### ISSUE TITLE
jrc_mus-kidney-2

### ISSUE THREAD

**Issue by jonesa3** [janelia-cellmap/cellmap-nuclei#24]

**Comment by avweigel** [comment]
@davidackerman path of tif that needs relabeling is under `3D Prediction Path - v0 (raw)`

Is this something that @jonesa3 could learn to do?

**Comment by davidackerman** [comment]
yeah for sure she can learn to do it! can you point me to the full path?

**Comment by jonesa3** [comment]
proofread tif found here: /avetissiane/tiff_slices/jrc_mus-kidney-2/s4_cp_masks_amira_proofread.tif

**Comment by yuriyzubov** [comment]
/avetissiane/tiff_slices/jrc_mus-kidney-2/s4_cp_masks_amira_proofread_relabel.zarr

**Comment by rinva** [comment]
![Image](https://github.com/user-attachments/assets/486edf1f-8467-4879-bb62-eb6fac52e7ab)

Thumbnail
/cellmap/nrs/data/jrc_mus-kidney-2/thumbnail_nuc.jpg

**Comment by rinva** [comment]
DOIs created
model - https://doi.org/10.25378/janelia.26506354	https://figshare.com/s/aa9bf7b4f9d7bfe9d04f
refined segmentations - https://doi.org/10.25378/janelia.26506534	https://figshare.com/s/e5f428be9f0a11c4c892

**Comment by rinva** [comment]
Ready for meshes.
